### ISSUE TITLE
Combination of disallowLockDaysInRange with lockDaysInclusivity doesn't work properly. 

### ISSUE THREAD

**Issue by mijrs** [wakirin/Litepicker#175]
**Describe the bug**
Combination of `disallowLockDaysInRange: true` with `lockDaysInclusivity: "()"` doesn't work properly. 

**To Reproduce**
https://jsfiddle.net/Mijrs/f8jw1ybr/
I'm able to select 12-feb till 19-feb in this example.

**Expected behavior**
Prevent to select date ranges with locked dates, also with lockDaysInclusivity.

**Comment by wakirin** [comment]
From the docs:
> **lockDaysInclusivity**
> A `[` indicates inclusion of a value. A `(` indicates exclusion. If the inclusivity parameter is used, both indicators must be passed.

This means, for example, that you have locked this range: `['2021-02-12', '2021-02-19']`

with `lockDaysInclusivity: "[]"` result will be:
```
(2021-02-12) - ... - (2021-02-19)
 from date^            to date^
```

with `lockDaysInclusivity: "()"` result will be:
```
(2021-02-12) - (2021-02-11)  -  ... - (2021-02-18) - (2021-02-19)
   ignore^      from date^              to date^        ignore^
```


Do not use `lockDaysInclusivity:" () "` unless you want to allow 12-feb and 19-feb selection.

**Comment by mijrs** [comment]
But selecting 11-feb till 20-feb is correctly not possible to select in my example, because contains a locked period.

The periode 12-feb till 19-feb also includes a locked period (for example locked day 15-feb), so therefore it shouldn't be possible.

**Comment by wakirin** [comment]
Ah, I see the problem. 
Thanks for the clarification.

I will fix it in next release.

**Comment by wakirin** [comment]
fixed in 2.0.2
